src/app/assess/wizard/key-code.enum.ts

```diff
@@ -0,0 +1,16 @@
+export enum KEY_CODE {
+    RIGHT_ARROW = 39,
+    LEFT_ARROW = 37,
+    UP_ARROW = 38,
+    DOWN_ARROW = 40,
+
+    TILDE = 192,
+    BACKTICK = 192,
+
+    ZERO = 48,
+    ONE = 49,
+    TWO = 50,
+    THREE = 51,
+    FOUR = 52,
+    FIVE = 53,
+}
```

Comment:
could pull in ts-keycode-enum?

Reply:
yeah ok, ill use that, I updated this PR